ui/src/reactions.ts

```diff
@@ -1,46 +0,0 @@
-import {StoreMapping} from './inject';
-import {reaction} from 'mobx';
-import * as Notifications from './snack/browserNotification';
-
-export const registerReactions = (stores: StoreMapping) => {
-    const clearAll = () => {
-        stores.messagesStore.clearAll();
-        stores.appStore.clear();
-        stores.clientStore.clear();
-        stores.userStore.clear();
-        stores.wsStore.close();
-    };
-    const loadAll = () => {
-        stores.wsStore.listen((message) => {
-            stores.messagesStore.publishSingleMessage(message);
-            Notifications.notifyNewMessage(message);
-            if (message.priority >= 4) {
-                const src = 'static/notification.ogg';
-                const audio = new Audio(src);
-                audio.play();
-            }
-        });
-        stores.appStore.refresh();
-    };
-
-    reaction(
-        () => stores.currentUser.loggedIn,
-        (loggedIn) => {
-            if (loggedIn) {
-                loadAll();
-            } else {
-                clearAll();
-            }
-        }
-    );
-
-    reaction(
-        () => stores.currentUser.connectionErrorMessage,
-        (connectionErrorMessage) => {
-            if (!connectionErrorMessage) {
-                clearAll();
-                loadAll();
-            }
-        }
-    );
```

Comment:
This seems to be not migrated into redux.

Reply:
Could you please explain what this code is doing or better what is the propose for it?

I do not see a reason, why you want to destroy your complete local store if the server is not available.
The store is destroyed if the user got invalid.

Reply:
If the client loses connection, then the client doesn't receive new messages via the websocket connection. When the connection is reestablished, we have to refetch everything to get an up-to-date state.